lib/src/modules/task.cc

```diff
@@ -11,6 +11,10 @@
 #include <boost/system/error_code.hpp>
 
 #include <rapidjson/rapidjson.h>
+#if RAPIDJSON_MAJOR_VERSION >= 1 && RAPIDJSON_MINOR_VERSION >= 1
```

Comment:
I'd suggest changing the condition to:
```c++
#if RAPIDJSON_MAJOR_VERSION > 1 || RAPIDJSON_MAJOR_VERSION == 1 && RAPIDJSON_MINOR_VERSION >= 1
```
as the original condition is not satisfied for x.0 (2.0, 3.0, etc.).

Reply:
Fixed.